ambry-tools/src/main/java/com.github.ambry/store/StoreCopier.java

```diff
@@ -270,6 +270,10 @@ public void close() throws IOException {
             for (Transformer transformer : transformers) {
               message = transformer.transform(message);
```

Comment:
there can be multiple transformers in the pipeline. A null check should go in here and break early IMO

Reply:
good idea, added